src/map/ai/controllers/mob_controller.cpp

```diff
@@ -342,7 +342,7 @@ bool CMobController::MobSkill(int wsList)
 
         PActionTarget = luautils::OnMobSkillTarget(PActionTarget, PMob, PMobSkill);
 
-        if (PActionTarget && !PMobSkill->isTwoHour() && luautils::OnMobSkillCheck(PActionTarget, PMob, PMobSkill) == 0) // A script says that the move in question is valid
+        if (PActionTarget && !PMobSkill->isAstralFlow() && luautils::OnMobSkillCheck(PActionTarget, PMob, PMobSkill) == 0) // A script says that the move in question is valid
```

Comment:
Is this "isAstralFlow" even needed here? Will anything even attempt to use an AF ability through this method?

Reply:
no idea, I just renamed the bastid. for now, revisit to murder the F outa it later coz the answer -should- be no but you never know whats flagged or being checked for the glag.